integration/maptest/map.go

```diff
@@ -177,55 +177,58 @@ func RunMapRevisionZero(ctx context.Context, t *testing.T, tadmin trillian.Trill
 		},
 	} {
 		for _, hsr := range tc.hashStrategy {
-			tree, hasher, smr, err := newTreeWithHasher(ctx, tadmin, tmap, hsr.hashStrategy)
-			if err != nil {
-				t.Errorf("%v: newTreeWithHasher(%v): %v", tc.desc, hsr.hashStrategy, err)
-			}
-			if got, want := smr.RootHash, hsr.wantRoot; want != nil && !bytes.Equal(got, want) {
-				t.Errorf("%v: newTreeWithHasher() returned unexpected root hash %x, want %x", tc.desc, got, want)
-			}
-			pubKey, err := der.UnmarshalPublicKey(tree.GetPublicKey().GetDer())
-			if err != nil {
-				t.Errorf("%v: UnmarshalPublicKey(%v): %v", tc.desc, hsr.hashStrategy, err)
-			}
+			t.Run(fmt.Sprintf("%v/%v", tc.desc, hsr.hashStrategy), func(t *testing.T) {
+				tree, hasher, smr, err := newTreeWithHasher(ctx, tadmin, tmap, hsr.hashStrategy)
+				if err != nil {
+					t.Fatalf("newTreeWithHasher(%v): %v", hsr.hashStrategy, err)
+				}
+				if got, want := smr.GetRootHash(), hsr.wantRoot; want != nil && !bytes.Equal(got, want) {
+					t.Fatalf("newTreeWithHasher() returned unexpected root hash %x, want %x", got, want)
+				}
+				pubKey, err := der.UnmarshalPublicKey(tree.GetPublicKey().GetDer())
+				if err != nil {
+					t.Fatalf("UnmarshalPublicKey(%v): %v", hsr.hashStrategy, err)
+				}
+
+				getSmrResp, err := tmap.GetSignedMapRoot(ctx, &trillian.GetSignedMapRootRequest{
+					MapId: tree.TreeId,
+				})
+				if err != nil {
+					t.Fatalf("GetSignedMapRoot(): %v", err)
+				}
+				if err := verifyGetSignedMapRootResponse(getSmrResp.GetMapRoot(), tc.wantRev,
+					pubKey, hasher, tree.TreeId); err != nil {
+					t.Errorf("verifyGetSignedMapRootResponse(rev %v): %v", tc.wantRev, err)
+				}
+				//
```

Comment:
What are these blank comment lines for, would a blank line suffice?

Reply:
Not sure. I'll remove it. I think it was pre-existing.